src/kusto/azext_kusto/tests/__init__.py

```diff
@@ -34,15 +39,33 @@ def get_func_to_call():
         func_to_call = func
         try:
             func_to_call = import_manual_function(func)
+            print("Found manual override for {}(...)".format(func.__name__))
         except (ImportError, AttributeError):
             pass
         return func_to_call
 
     def wrapper(*args, **kwargs):
         func_to_call = get_func_to_call()
         print("running {}()...".format(func.__name__))
-        return func_to_call(*args, **kwargs)
+        try:
+            return func_to_call(*args, **kwargs)
+        except (AssertionError, AzureError, CliTestError, CliExecutionError, JMESPathCheckAssertionError) as e:
+            print("--------------------------------------")

```

Comment:
We should use logger instead of print.

Reply:
@changlong-liu Can you take a look?

Reply:
The print is generated by autorest.az.  
I just created an issue here for it: https://github.com/Azure/autorest.az/issues/497. and will work on it.

Reply:
Sure. Let's solve this issue in another PR.